lmdeploy/serve/async_engine.py

```diff
@@ -391,6 +392,7 @@ def batch_infer(
                 True, which means chat_template will be applied.
             adapter_name (str): the adapter name of slora for pytorch backend.
                 Pick one from adapters. Default to None, using the base model.
+            progress_bar (tqdm): the progress bar used in pipeline benchmark
```

Comment:
can we make it a true or false argument? May update `__call__` function as well.

Reply:
Sure. That's a good idea

Reply:
fixed